<launch>

    <!-- Start the frontier exploration client -->
    <node pkg="frontier_exploration" type="explore_client" name="explore_client" output="screen" />


    <!-- Start the frontier exploration server -->
    <node pkg="frontier_exploration" type="explore_server" name="explore_server" output="screen">
        <param name="frequency" value="1.0" />

        <!-- Maximum goal distance between each other -->
        <param name="goal_aliasing" value="2.0" />

        <!-- load the costmap of the planner -->
        <rosparam file="$(find markhor_navigation)/config/costmap_common.yaml" command="load"
            ns="explore_costmap" />

        <!-- load the exploration costmap -->
        <rosparam file="$(find markhor_navigation)/config/costmap_exploration.yaml" command="load"
            ns="explore_costmap" />
    </node>
</launch>